### DISCUSSION TITLE
Ability to search tags

### DISCUSSION THREAD

**Discussion by qanadin** [mealie-recipes/mealie#2489] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Having the search bar have the ability to use search bar to search recipes based on tags.

### Please provide a concise description of the feature that would resolve your issue.

If I type soup, I want it to search for all recipes that have the tag soup in them.  This would allow for quick pull of all my soup recipes. 

### Please consider and list out some caveats or tradeoffs made in your design decision

Possibly removing the tag button below the search bar, where you can search in there.  This is something for those user who aren't all that adept to finding this ability on their own.   

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
There are multiple ways to search by tags now, either using the dedicated tags page or selecting the tags filter
